Lib/fractions.py

```diff
@@ -62,7 +62,7 @@ def _hash_algorithm(numerator, denominator):
     (?:                                   # followed by
        (?:\s*/\s*(?P<denom>\d+(_\d+)*))?  # an optional denominator
     |                                     # or
-       (?:\.(?P<decimal>d*|\d+(_\d+)*))?  # an optional fractional part
+       (?:\.(?P<decimal>\d*|\d+(_\d+)*))? # an optional fractional part
```

Comment:
Nit: we need to shift all those comments right one column, to satisfy PEP8's "Inline comments should be separated by at least two spaces from the statement." rule.

Reply:
Alright, gotcha. I could make a news entry.